### ISSUE TITLE
Support exporting excalidraw files

### ISSUE THREAD

**Issue by zhyma** [KosmosisDire/obsidian-webpage-export#108]
**Describe the bug**
I'm not able to export excalidraw files.
- If I try to export the whole folder, It will generate an error (please see the error log attached to the last) and refused to continue.
- If I try to export an individual file, it sometimes causes the whole Obsidian becomes black. I can only close and reopen it. The generated file only shows "page not found"

Obsidian version 1.2.8, Webpage HTML Export version 1.5.2, Excalidraw version 1.9.1 (the draw files are created by a previous version)
The include Plugin CSS - Excalidraw is selected.

**Operating System**
Windows 10

**To Reproduce**
The excalidraw file has a PNG embedded (copied and pasted into the excalidraw) and some handwriting over the PNG image.

##### [Warning] Cannot render file: 
```
This file was not a normal markdown file! File: 1.theory/robot_basics/zz.handwritten/z.Drawing 2022-01-18 10.08.31.excalidraw.md
```

##### [Error] Could not export file: z.Drawing 2022-01-18 10.08.31.excalidraw.md
```
Error: vaultRelativeDestination must be a folder
    at new Downloadable (plugin:webpage-html-export:10600:13)
    at ExportFile.getSelfDownloadable (plugin:webpage-html-export:12162:12)
    at HTMLGenerator.generateWebpage (plugin:webpage-html-export:11723:33)
    at async HTMLExportPlugin.exportFile (plugin:webpage-html-export:12306:7)
    at async HTMLExportPlugin.exportFolder (plugin:webpage-html-export:12346:28)
    at async eval (plugin:webpage-html-export:12268:30)
```

**Comment by KosmosisDire** [comment]
I think the error is pretty self explainatory. You cannot export excalidraw files because they are not normal markdown files. This may change in the future but for now there is no way to export excalidraw files. You can embed the excalidraw inage into a note and export that note though.

**Comment by KosmosisDire** [comment]
Also as far as the errors, those are caused by this issue: #94

**Comment by cmorg789** [comment]
Typescript isn't even close to any language I've used before, but I think I found a bit that would be useful. Excalidraw has an API defined in [ExcalidrawAutomate.ts](https://github.com/zsviczian/obsidian-excalidraw-plugin/blob/master/src/ExcalidrawAutomate.ts)

createSVG:
```
async createSVG(
    templatePath?: string,
    embedFont: boolean = false,
    exportSettings?: ExportSettings, 
    loader?: EmbeddedFilesLoader,
    theme?: string,
    padding?: number,
  ): Promise<SVGSVGElement> 
```
getTemplate:
```
async function getTemplate(
  plugin: ExcalidrawPlugin,
  fileWithPath: string,
  loadFiles: boolean = false,
  loader: EmbeddedFilesLoader,
  depth: number
): Promise<{
  elements: any;
  appState: any;
  frontmatter: string;
  files: any;
  hasSVGwithBitmap: boolean;
}>
```

**Comment by KosmosisDire** [comment]
This should be implemented in v1.7.1! :) Let me know if you have any problems with it
